src/main/java/jp/ac/keio/bio/fun/xitosbml/image/SpatialImage.java

```diff
@@ -322,15 +336,28 @@ public void setAdjacentsList(ArrayList<ArrayList<String>> adjacentsList) {
 	 * @param path the path to the directory
 	 * @param name the name of TIFF file
 	 */
-	public void saveAsImage(String path, String name){
+	public void saveAsImage(String path, String name) {
 		FileSaver fs = new FileSaver(img);
-		if(name == null) return;
-		if(depth > 1)
+		if (name == null)
+			return;
+		if (depth > 1)
 			fs.saveAsTiffStack(path + "/" + name + ".tiff");
 		else
 			fs.saveAsTiff(path + "/" + name + ".tiff");
 	}
 
+	/**
+	 * Save image as TIFF file --> for the CUI version
+	 * 
+	 * @param path2
+	 */
+	public void saveAsImageCui(String path2) {
+		FileSaver fs = new FileSaver(img);
+		if (path2 == null)
+			return;
+		fs.saveAsTiff(path2 + "_sptl" + ".tiff");
```

Comment:
Why do we save the imager as `"path2"_sptl.tiff` instead of `"path2".tiff`?

Reply:
The only difference between path1 and path2 on my end was the difference in extension, i.e. .tif and .xml. Since I did not want the original image to be overwritten, I saved imager as path2_sptl.tiff instead of path2.tiff.

Reply:
Ok, that make sense.  Then why don't we check if `"path2".tiff` exist first and if there is save it as `"path2"_sptl.tiff` otherwise save it as `"path2".tiff`.

Reply:
Sure, but how to check whether 'path2.tiff' exists in the same directory?

Reply:
Maybe use `java.io.file`?
https://docs.oracle.com/javase/9/docs/api/java/io/File.html#exists

Reply:
Done. Please refer to https://github.com/Medha-B/XitoSBML/issues/6.